docs/FAQ.md

```diff
@@ -23,6 +23,9 @@ We have reports of success on the following terminal emulators:
 * Microsoft/Windows Terminal (https://github.com/Microsoft/Terminal) **Windows only**
 * mosh (https://mosh.org/)
 
+Among the above, minor non-breaking issues have been noticed with the following:
+* Microsoft/Windows Terminal **Windows Only** - Bold text isn't actually bold, it either renders the same as normal text or renders in a different colour (https://github.com/microsoft/terminal/issues/109)
```

Comment:
Does this deserve its own section separate from the issues section below it?

Reply:
The reason I didn't fit it in the issues section below is because that section only has emulators that we consider unusable and not among the successful emulators. Hence putting Windows Terminal in the successful emulators section as well as the issues section seemed inappropriate. This issue also seemed too minor to take it off of the successful emulators list.